recipes/duckx/all/conanfile.py

```diff
@@ -0,0 +1,83 @@
+from conan import ConanFile
+from conan.errors import ConanInvalidConfiguration
+from conan.tools.files import apply_conandata_patches, export_conandata_patches, copy, get, rmdir
+from conan.tools.cmake import CMake, CMakeDeps, CMakeToolchain, cmake_layout
+from conan.tools.build import check_min_cppstd
+import os
+
+required_conan_version = ">=1.53.0"
+
+class DuckxConan(ConanFile):
+    name = "duckx"
+    description = " C++ library for creating and updating Microsoft Word (.docx) files."
+    license = "MIT"
+    url = "https://github.com/conan-io/conan-center-index"
+    homepage = "https://github.com/amiremohamadi/DuckX/"
+    topics = ("docx", "docx-files", "office")
+    settings = "os", "arch", "compiler", "build_type"
+    options = {
+        "shared": [True, False],
+        "fPIC": [True, False],
+    }
+    default_options = {
+        "shared": False,
+        "fPIC": False,
+        "pugixml/*:header_only": True,
+    }
+
+    @property
+    def _min_cppstd(self):
+        return 11
+
+    def export_sources(self):
+        export_conandata_patches(self)
+
+    def config_options(self):
+        if self.settings.os == "Windows":
+            del self.options.fPIC
+
+    def configure(self):
+        if self.options.shared:
+            self.options.rm_safe("fPIC")
+
+    def layout(self):
+        cmake_layout(self, src_folder="src")
+
+    def requirements(self):
+        self.requires("pugixml/1.14", transitive_headers=True)
+        self.requires("kuba-zip/0.2.6", transitive_headers=True)
```

Comment:
```suggestion
        self.requires("kuba-zip/0.3.1", transitive_headers=True)
```
Maybe we can try and bump this while we're at it